ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/JdbcLedgerDao.scala

```diff
@@ -1056,7 +1056,9 @@ private class JdbcLedgerDao(
                   "package" -> p._1.toByteArray
               )
             )
-          val updated = executeBatchSql(dbType.SQL_INSERT_PACKAGE, params).map(math.max(0, _)).sum
+          val updated = {
+            executeBatchSql(dbType.SQL_INSERT_PACKAGE, params).map(math.max(0, _)).sum
```

Comment:
Not very important, but this looks like a leftover from some iteration in which you performed some extra operation along with the batch execution for debugging purposes. :slightly_smiling_face:

Reply:
Thanks ;-)